jobs/furnishings/flags.json

```diff
@@ -2,6 +2,19 @@
     "flagValues": {
         "enable-involuntary-dissolution": true,
         "disable-dissolution-sftp-bcmail": true,
-        "disable-dissolution-sftp-bclaws": false
+        "disable-dissolution-sftp-bclaws": false,
+        "db-versioning": {
+            "legal-api": true,
+            "emailer": false,
+            "filer": false,
+            "entity-bn": false,
+            "digital-credentials": false,
+            "dissolutions-job": false,
+            "furnishings-job": false,
```

Comment:
That would be nice if you could provide some local testing examples to verify if the versioning switches work correctly through the flag of furnishings-job